ida/ida_import.py

```diff
@@ -143,4 +153,4 @@ def main(json_file):
 
 
 if __name__ == '__main__':
-    main(ida_kernwin.ask_file(1, '*.json', 'Import file name'))
+    main(ida_kernwin.ask_file(0, '*.json', 'Import file name'))
```

Comment:
The file explorer popup was asking me to save the file when importing, I assume this is a mistake so I changed the param here.